src/SDKs/Batch/Management/Management.Batch/Microsoft.Azure.Management.Batch.csproj

```diff
@@ -2,16 +2,24 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Batch</PackageId>
-    <Description>Provides management capabilities for Azure Batch service accounts.</Description>
+    <Description>Provides management functions for Microsoft Azure Batch services.</Description>
     <AssemblyTitle>Microsoft Azure Batch Management Library</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.Batch</AssemblyName>
     <VersionPrefix>5.1.0</VersionPrefix>
-    <PackageTags>Microsoft Azure batch management</PackageTags>
+    <PackageTags>Microsoft Azure Batch Management Library</PackageTags>
```

Comment:
Is library really required here?

Reply:
maybe get rid of that and add "windowsazureofficial"

Reply:
Fixed.